qa/suites/fs/functional/tasks/snap_schedule_snapdir.yaml

```diff
@@ -0,0 +1,30 @@
+overrides:
+  ceph:
+    conf:
+      mgr:
+        debug mgr: 20
+        debug ms: 1
+        debug finisher: 20
+        debug client: 20
+    log-whitelist:
+      - OSD full dropping all updates
+      - OSD near full
+      - pausewr flag
+      - failsafe engaged, dropping updates
+      - failsafe disengaged, no longer dropping
+      - is full \(reached quota
+      - POOL_FULL
+      - POOL_BACKFILLFULL
+
+tasks:
+  - cephfs_test_runner:
+      modules:
+        - tasks.cephfs.TestSnapSchedulesSnapdir
+
+overrides:
+  kclient:
+    snapdirname: .customsnapkernel
```

Comment:
You didn't check this in your test code below.

Reply:
Have added check for kclient.